sklearn/metrics/cluster/tests/test_supervised.py

```diff
@@ -172,6 +172,20 @@ def test_expected_mutual_info_overflow():
     assert expected_mutual_information(np.array([[70000]]), 70000) <= 1
 
 
+def test_int_overflow_mutual_info_score():
+    # Test overflow in mutual_info_classif
+    x = np.concatenate((np.repeat(1, 52632 + 2529), np.repeat(2, 14660+793),
+                       np.repeat(3, 3271+204), np.repeat(4, 814+39),
```

Comment:
put space between the arithmetic signs

Reply:
I think that this is easier to read `np.array([0] * 10000 + [1] * 10000 + [2] * 10000)` than the call with `np.repeat`.
You can also make the sum directly instead of making the addition.